### ISSUE TITLE
Organization Logo

### ISSUE THREAD

**Issue by Mottie** [rejuvenate/rejuvenate#2]
I threw together an organization logo... let me know if you guys like it and if so I'll add a folder to this repo with the image and svg.

![rejuvenate-logo](https://cloud.githubusercontent.com/assets/136959/22747936/0b1fd7fc-edee-11e6-8c4b-ceb86233a4dd.png)

**Comment by ThomDietrich** [comment]
I like it! I've also thought about a logo and came up with this yesterday (but was not happy yet):

![inkscape](https://cloud.githubusercontent.com/assets/2870104/22751556/1c41d1e2-ee35-11e6-8367-9a4a4e6fa4ab.png)

Yours might a bit too "un-square". Maybe you could turn it by 45° to better fit into the avatar square?

**Comment by Mottie** [comment]
I could make it more round... LOL

**Comment by Mottie** [comment]
Here are some tweaked versions...

| | |
|----------|-----------|
| Original Rotated |  ![rotated](https://cloud.githubusercontent.com/assets/136959/22755184/ac71b486-ee07-11e6-9314-9fc055c82235.png) |
| Rounded | ![rounded](https://cloud.githubusercontent.com/assets/136959/22755192/b570ed0e-ee07-11e6-87a9-304e6dfe3b48.png) | 
| Rounded & Rotated | ![rounded-rotated](https://cloud.githubusercontent.com/assets/136959/22755200/c04ec5fc-ee07-11e6-87f5-5b6ad959c0e3.png) |

It was originally narrow to make the logo seem 3D, so the rounded ones don't seem quite right to me since I didn't make them perfectly round... 

I like the rotated one the best.

**Comment by ThomDietrich** [comment]
I like the "Original Rotated". @TPS ?

**Comment by TPS** [comment]
@Mottie I agree w/ @ThomDietrich (the "Original Rotated" is the best so far) but had a further idea: In keeping w/ the Rejuvenate overall theme, could that be mixed a little w/ "♻️" in some lively Spring Green color blended in?  I think that'd really be ideal.

**Comment by Mottie** [comment]
Do you mean making the "j" a little brighter green? I tried to make the "r" more like the sky and the "j" more grass colored.

**Comment by TPS** [comment]
Heh, I ___completely___ missed the "rj" but now it seems obvious.  Since I didn't see that, the similarity to the ♻️ symbol is what I saw, but wanted to push it further in that direction, along with blending either the blue or gray smoothly into Spring Green.

**Comment by Mottie** [comment]
I'm not sure what you mean... are you saying use a gradient? I originally had a gradient of colors, but decided on simplifying the design. It currently has six solid colors.

If I go ahead and add the logo files, maybe you can mess around with them and share your changes here? I am completely open to any ideas and suggestions.

**Comment by Mottie** [comment]
Done: https://github.com/rejuvenate/rejuvenate/tree/master/logo

**Comment by TPS** [comment]
Yes, _that's_ what I meant: a gradient from, say, blue to green, or the gray saturating into green.  I'll see if I can fiddle the SVG a bit on the weekend.  Thanks for posting the sources, @Mottie. _(Sorry about the random auto-**in**correction. :bow:)_

**Comment by Mottie** [comment]
Here is the earlier iteration I mentioned... 

![rejuvenate-gradient](https://cloud.githubusercontent.com/assets/136959/22771624/7e6fbbea-ee5d-11e6-9da9-0c566b802786.png)

Personally, I don't like it as much.

**Comment by ThomDietrich** [comment]
Ooh :see_no_evil: "rj"... I did also not see that. That's a great idea!!
Can you think of a way to hint that a bit? maybe by adding the tick of the "r" and the dot of the "j" somehow...? I'll add the logo as it is right now as the avatar, we can still exchange it later. Thanks @Mottie !

**Comment by TPS** [comment]
I actually prefer that gradient version (https://github.com/rejuvenate/rejuvenate/issues/2#issuecomment-278557611), but let's see, maybe, more vibrant green (🍏💚)  improves it.  I'll start from that 1 on weekend,  if you could upload all versions of that, also, @Mottie.

**Comment by Mottie** [comment]
> Can you think of a way to hint that a bit? maybe by adding the tick of the "r" and the dot of the "j" somehow...?

I think it's cooler that it's subtle
<img title=":_tuzki_teehee:" src="https://cloud.githubusercontent.com/assets/136959/13203716/665ac5c2-d883-11e5-87ef-bbb0be4e1d81.gif">

**Comment by ThomDietrich** [comment]
Okay also fine with me. Let's go back to the more pressing issues (#1)

**Comment by TPS** [comment]
Unfortunately, you're _too_ subtle for me, @Mottie, but that's _my_ fault/problem. 🤓😲

**Comment by TPS** [comment]
Something like ![Gradient lightened Rejuvenate logo](https://github.com/rejuvenate/rejuvenate/raw/master/logo/TPS/rejuvenate-logo-gradient-small-palletted-tweaked.png) is what I meant.  I uploaded all my versions to  `/logo/TPS/`, because I'm not vaguely proficient at this, so there's problems, ~~such as transparency,~~ but I wanted to _show_ the Rejuvenat[ion] in the logo moreso.

Edit: I realized I could do better. 🎉 :smile:

**Comment by TPS** [comment]
@rejuvenate/community Since y'all have seem to gotten a chance to look over this way, what do think of [my tweak to @Mottie's logo](https://github.com/rejuvenate/rejuvenate/issues/2#issuecomment-279200361)?:
>Something like ![Gradient lightened Rejuvenate logo](https://github.com/rejuvenate/rejuvenate/raw/master/logo/TPS/rejuvenate-logo-gradient-small-palletted-tweaked.png) is what I meant.… I wanted to _show_ the Rejuvenat[ion] in the logo moreso.

**Comment by Mottie** [comment]
I'm out of the country with only my cell until next week... the colors need some tweaking because the green gradient should be the reverse of what you see now.

**Comment by TPS** [comment]
No rush, @Mottie.  When you get back in town, fiddle the source files I modified & uploaded the way you like, so as to show me what you mean by "the green gradient should be the reverse of what you see now." 🤔

**Comment by Mottie** [comment]
Well, here is my gradient version... it's not perfect 😞 

![rejuvenate-logo5-gradient](https://cloud.githubusercontent.com/assets/136959/23868729/82588930-07ee-11e7-8dde-3531178bf616.png)

I spoke too soon about reversing the green gradient.. things on my phone are tiny.

<img title=":_ onion_head_lol2 :" src="https://cloud.githubusercontent.com/assets/136959/13371599/b87a55d2-dcef-11e5-9640-02a14877cdb3.gif">

**Comment by TPS** [comment]
It's not bad, @ all!  From your source SVG, I started w/ something like that, but wanted to brighten it up (inspired by sunlit spring colors after a good rain, which I'm currently missing dreadfully), so that's where mine came from.  Also, I like Escher stuff, so I lucked into a tiny bit of that. 😁
